container_test.go

```diff
@@ -629,6 +632,77 @@ func TestContainerKill(t *testing.T) {
 	}
 }
 
+func TestKillContainerDeletedByRunc(t *testing.T) {
+	t.Parallel()
+
+	// We skip this case when runtime is crun.
```

Comment:
could you mind to add `More information in https://github.com/containerd/containerd/pull/4214#discussion_r422769497` in other line? thanks

Reply:
OK, and should I open issue on oci container-spec for runtime exception?

Reply:
Sure.